src/app/[locale]/settings/layout.tsx

```diff
@@ -61,12 +61,12 @@ export default async function SettingsLayout({
           </div>
         </div>
         <Separator className='my-6' />
-        <div className='flex h-full w-full flex-col space-y-8 lg:flex-row lg:space-x-12 lg:space-y-0'>
+        <div className='flex h-full w-full flex-col gap-8 lg:flex-row'>
           <NextIntlClientProvider
             messages={{ settings, ui } as Pick<Messages, 'settings' | 'ui'>}
           >
-            <aside className='-mx-4 -my-2 lg:my-0 lg:w-1/5'>
-              <SidebarNav showAdministratorMenu={showAdministratorMenu} />
+            <aside className='-my-2 w-full lg:my-0 lg:w-1/5'>
+              <SidebarNav showAdministratorMenu={true} />
```

Comment:
This seems like an unintentional change from when you were testing
```suggestion
            <aside className='-my-2 w-full lg:my-0 lg:w-1/5'>
              <SidebarNav showAdministratorMenu={showAdministratorMenu} />
```